i18n/validate.py

```diff
@@ -46,8 +46,18 @@ def validate_po_files(configuration, locale_dir, root_dir=None, report_empty=Fal
                 # check for empty translations. But don't check English.
                 if "/locale/en/" not in filename:
                     problems = check_messages(filename, report_empty)
-                    found_problems = problems or found_problems
-                    report_problems(filename, problems)
+                    if problems:
+                        report_problems(filename, problems)
+                        found_problems = True
+
+                    dup_filename = filename.replace('.po', '.dup')
+                    has_duplicates = os.path.exists(dup_filename)
+                    if has_duplicates:
+                        log.warning(" Duplicates found in %s, details in .dup file", dup_filename)
+                        found_problems = True
+
+                    if not (problems or has_duplicates):
+                        log.info(" No problems found in %s", filename)
```

Comment:
1. Remove the preceding space.
2. Pick a quote style—single or double—and stick with it.